src/SDKs/_metadata/network_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-04-26 20:59:33 UTC
+2018-05-15 11:02:49 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
-Branch:      master
-Commit:      b9721ae4c9f04914cc89260b8339dc33372f835b
+Branch:      Network-2018-05-01
```

Comment:
@sayghosh We cannot accept this PR. The SDK MUST be generated from `master` branch of REST specs repo.
Please coordinate with @MikhailTryakhov to create a PR for the May branch and integrate your changes there. `